sequencer/api/migrations/postgres/V403__drop_undecided_state.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE undecided_state;
```

Comment:
why does this migration has V403?

Reply:
sorry but why this migration has changed from V503 to V403?

Reply:
because it's happening before the V5XX migrations hit